src/components/app-configure/chain-management/AppChainManagementForm.vue

```diff
@@ -45,11 +47,17 @@ const enableSave = computed(() => {
     chainType.length &&
     currency.length &&
     name.length &&
-    isValidUrl(explorerURL.trim()) &&
+    (explorerURL.length ? isValidUrl(explorerURL.trim()) : true) &&
```

Comment:
Can this also be explorerURL.trim().length?

Reply:
Or the whole condition can be

(!explorerURL.trim().length || isValidUrl(explorerURL.trim())